README.md

```diff
@@ -71,7 +98,8 @@ To manage dependancy on the backend, we are using [poetry](https://python-poetry
 | DB_USER | User of the DB | uquizz |
 | DB_NAME | Name of the DB | uquizz |
 | DB_PASSWORD | Password of the DB | uquizz |
-| BACK_PORT | External port back access | 8000 |
+| BACK_PORT | External back port access | 8000 |
+| BACK_PORT | External front port access | 8080 |
```

Comment:
A little typo here, I think you meant FRONT_PORT ;)

Reply:
right !